packages/components/src/components/dropdown/dropdown.jsx

```diff
@@ -370,17 +375,41 @@ const Dropdown = ({
                         id='dropdown-display'
                         ref={dropdown_ref}
                     >
-                        <DisplayText
-                            has_symbol={has_symbol}
-                            name={name}
-                            is_title={is_list_visible}
-                            placeholder={placeholder}
-                            value={value || 0}
-                            list={list}
-                            is_align_text_left={is_align_text_left}
-                        />
+                        {is_filter && (
```

Comment:
Let's make this more general instead of just a filter `Icon` added to Dropdown?
Can have the option of a `suffix_icon` and pass it as a `prop` to give us more flexibility. 🙏🏻